### PR TITLE
APIs for tab CRUD in chats

### PR DESCRIPTION
**Pull request by AkJo**
This PR deals with introduction of tab CRUD APIs for chats, and fixes inconsistencies in the tab crud APIs for channels.

(Note: This PR is based on another pending PR branch. That is the reason for the extra files in this PR).

### CODE REVIEW

File: api-reference/beta/api/chat-patch-tabs.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: "Update tab in chat"
```
Comment (Lauragra) [inline api-reference/beta/api/chat-patch-tabs.md:2]: Does the tool output file names with patch? The standard convention is to use update in the file names.

File: api-reference/beta/api/chat-patch-tabs.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+|:---------------|:--------|
+| Authorization  | Bearer {token}. Required.  |
+| Content-Type  | application/json  |
+
+## Request body
+In the request body, supply a JSON representation of [tab](../resources/teamstab.md) object.
+
+## Response
+
+If successful, this method returns a `200 OK` response code.
```
Comment (Lauragra) [inline api-reference/beta/api/chat-patch-tabs.md:46]: Update to state that it also returns an updated teamstab object, per the example.

File: api-reference/beta/api/chat-patch-tabs.md (hunk 1 above)
Comment (AkJo) [reply api-reference/beta/api/chat-patch-tabs.md:2]: Leaving it based as is based on discussion.

File: api-reference/beta/api/chat-patch-tabs.md (hunk 2 above)
Comment (AkJo) [reply api-reference/beta/api/chat-patch-tabs.md:46]: Fixed.

**Pull request merged**